src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -230,6 +230,7 @@ class Comment extends React.Component<Props, State> {
                         <div>
                             <ActivityTimestamp date={createdAtTimestamp} />
                         </div>
+                        <div className="bsc-Comment-separator" />
```

Comment:
1. Typo `bsc` -> `bcs`
2. This will likely affect the existing Preview experience; what does it look like?
3. We typically want to avoid elements that have no semantic value. Visual separation should be achieved with styles on existing elements.

Reply:
It gives the spacing between the Comment timestamp and message
ok, will remove the additional div and use the 

![Screenshot 2022-11-17 at 23 29 05](https://user-images.githubusercontent.com/8361509/202573705-1bcdebce-7a1d-41fc-b058-49dffd9249ba.png)

Reply:
Got it. Does it look correct when editing the comment?